docs/iris/src/whatsnew/3.0.3.rst

```diff
@@ -0,0 +1,587 @@
+.. include:: ../common_links.inc
+
+v3.0.3 (XX July 2021)
+*********************
+
+This document explains the changes made to Iris for this release
+(:doc:`View all changes <index>`.)
+
+
+.. dropdown:: :opticon:`alert` v3.0.3 Patches
+   :container: + shadow
+   :title: text-primary text-center font-weight-bold
+   :body: bg-light
+   :animate: fade-in
+   :open:
+
+   The patches included in this release include:
+
+   🐛 **Bugs Fixed**
+
+   #. `@lbdreyer`_ modified :meth:`~iris.cube.Cube.intersection` to use a tolerant 
+      equality check, when looking for cells that straddle the wrapping point.
+      (:pull:`4220`)
+
+
+.. dropdown:: :opticon:`alert` v3.0.2 Patches
+   :container: + shadow
+   :title: text-primary text-center font-weight-bold
+   :body: bg-light
+   :animate: fade-in
+   :open:
```

Comment:
@lbdreyer Could you remove this `:open:`.

For patches I've been only keeping the latest patch dropdown open by default, so that it is highlighted to the reader - and they can open the other dropdowns by choice.

<img width="386" alt="firefox" src="https://user-images.githubusercontent.com/2051656/124559236-6a1e4880-de33-11eb-84f4-8792bacce9b3.PNG">